tools/install-MXU.py

```
@@ -82,28 +86,12 @@ def install_resource():
         install_path,
     )
     
-    # MXU: 将 bbc_team_config_nomwu.json 复制并重命名为 bbc_team_config.json
-    nomwu_config = working_dir / "assets" / "bbc_team_config_nomwu.json"
+    # MXU: 复制 bbc_team_config.json（使用 scan_select 类型）
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)